plugins/org.jkiss.dbeaver.ext.generic/plugin.xml

```diff
@@ -755,6 +755,24 @@
                     <parameter name="supports-truncate" value="false"/>
                     <parameter name="supports-references" value="false"/>
                 </driver>
+                <driver
+                    id="kyuubi_hive"
+                    label="Apache Kyuubi (Incubating)"
+                    icon="icons/kyuubi_hive_icon.png"
+                    iconBig="icons/kyuubi_hive_icon_big.png"
+                    class="org.apache.kyuubi.jdbc.KyuubiHiveDriver"
+                    sampleURL="jdbc:hive2://{host}[:{port}][/{database}]"
+                    defaultPort="10009"
+                    description="Kyuubi Thrift Server connector"
+                    webURL="https://kyuubi.apache.org/"
+                    category="Hadoop"
+                    categories="hadoop">
+                    <file type="jar" path="maven:/org.apache.kyuubi:kyuubi-hive-jdbc-shaded:RELEASE" bundle="!drivers.kyuubi"/>
+                    <file type="jar" path="drivers/kyuubi" bundle="drivers.kyuubi"/>
```

Comment:
This is for cloudbeaver, I can send PR once this get merged.

<img width="1920" alt="image" src="https://user-images.githubusercontent.com/26535726/169705581-05e9a99c-edea-46f3-91f8-3acc863cc16f.png">

Reply:
I found that the cloud beaver community have concerns about the driver dependencies and size.
https://github.com/dbeaver/cloudbeaver/issues/178#issuecomment-772631295

I'd like to clarify that Kyuubi Hive JDBC driver is based on the Hive JDBC driver and we are keep improving it. Currently, we shaded and relocated all transitive classes except to Hadoop(for Kerberos) and SLF4J. And we have a plan to rewrite this Driver totally to make it light and zero-deps in the future.